.github/actions/numpy_vers/action.yml

```diff
@@ -62,7 +63,7 @@ runs:
             ;;
 
             # TODO: 'Windows*' might not be good
-            Windows*:x86_64)
+            MSYS_NT-10.0-17763:x86_64)
```

Comment:
Nit: `MSYS_NT-10.0-*:x86_64` maybe, 17763 is a build release, it's likely to break often

Reply:
Or even better `${TC_MSYS_VERSION}` since you define it